### PR TITLE
micrometer: adds configuration to skip tracing health

### PR DESCRIPTION
**Pull request by codefromthecrypt**
workaround to https://github.com/spring-projects/spring-boot/issues/34801 to prevent excessive health traces in zipkin

### CODE REVIEW

File: webflux6-micrometer/src/main/java/brave/example/Backend.java [hunk 1]
```diff
@@ -4,13 +4,15 @@
...
 import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.context.annotation.Import;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestHeader;
 import org.springframework.web.bind.annotation.RestController;
 import reactor.core.publisher.Mono;
 
 @EnableAutoConfiguration
 @RestController
+@Import(ObservationConfiguration.class)
```
Comment (reta) [inline webflux6-micrometer/src/main/java/brave/example/Backend.java:15]: Hm ... Could you please hint why you need it? The `spring-boot-actuator` is a dependency and `ObservationAutoConfiguration` should be picked by `@EnableAutoConfiguration`, no?

**Review by reta — APPROVED** [review]

File: webflux6-micrometer/src/main/java/brave/example/Backend.java (hunk 1 above)
Comment (codefromthecrypt) [reply webflux6-micrometer/src/main/java/brave/example/Backend.java:15]: should I change name to CustomObservationConfiguration?

Comment (codefromthecrypt) [reply webflux6-micrometer/src/main/java/brave/example/Backend.java]: done!

**Comment by codefromthecrypt** [comment]
thanks for the look folks. thanks also @ryanjbaxter for the workaround

**Pull request merged**